manager/src/static/js/marks.js

```diff
@@ -171,7 +186,7 @@ function addNewMark(mark, o, trail, svgCanvas, scale, show_telemetry) {
   var text = mark.text(0, 0, "");
   var foreignObject = document.createElementNS(
     "http://www.w3.org/2000/svg",
-    "foreignObject",
+    "foreignObject"
```

Comment:
Please keep the coding convention used in js codebase so far.

The rationale behind:
- Consistency: JS codebase is already using this convention, let's keep it consistent. Also, many style guides and tools (Prettier, ESLint) recommend trailing commas for multi-line objects and arrays.
- Cleaner diffs:
When adding or removing items, only the changed line is modified in version control, making diffs easier to read.
- Easier editing:
You can add new elements without worrying about whether to add or remove a comma from the previous line.


```suggestion
    "foreignObject",
```